include/Util/Color.hpp

```diff
@@ -23,6 +23,17 @@ class Color : public glm::vec4 {
     Color(const glm::vec4 &vec)
         : glm::vec4(vec) {}
 
+    /*
+     * @brief Parameterized constructor.
+     *
+     * @param r The red component of the color.
+     * @param g The green component of the color.
+     * @param b The blue component of the color.
+     * @param a The alpha component of the color.
+     */
+    Color(Uint8 r, Uint8 g, Uint8 b, Uint8 a = 255)
+        : glm::vec4(r, g, b, a) {}
+
```

Comment:
So we should delete constructors?

Reply:
im not sure, let's ask @NOOBDY

Reply:
I think it would be best if there is only one standard way to construct a `Color` object and the rest are helper functions, `Color` could even be a `struct` IMO

Reply:
I only chose `glm::vec4` as an implementation detail because it was convenient at the time, it could change if that suits you

Reply:
I will keep this implementation and close this conversation as resolved